docs/analytics_examples/warehouse_tutorial.md

```diff
@@ -33,7 +33,7 @@ The queries represented in the following tutorial are as follows:
 The tools that we can use to answer them are:
 * [**Metabase**](metabase) - our business insights and dashboarding tool
 * [**SQL**](querying-sql-jupyterhub) - using JupyterHub cloud notebooks
-* [**Python**](jupyterhub) - using JupyterHub cloud notebooks
+* [**Python**](jupyterhub-intro) - using JupyterHub cloud notebooks
```

Comment:
Do we have externally maintained links (in Google docs, slide decks, etc.) that are going to need updating from this?

Reply:
Hard to know for sure but unlikely, we tend to just link to the top-level page